### ISSUE TITLE
When will be the Next Release is it still active

### ISSUE THREAD

**Issue by sjkeerthi** [cachethq/cachet#4314]
Hi All,

Is there any plan to release  2.4 this year, after May 2019 there is not any update or release is it still active.

**Comment by nixmomo** [comment]
it dont look active now.... as you can see in "insights" the last commit is from march 2021

**Comment by elderlabs** [comment]
Going to assume the project is abandoned. A shame, really. Remains one of the best status-page softwares. Sure, it's open-source, but I'm no PHP wizard, and on the surface, this project seems quite complex to newcomers. All the fixes to common problems I'm seeing these days are along the lines of "use an outdated version of PHP that isn't available on your OS," among other things. ~~Migrated to nginx a while back and noticed Cachet isn't functional. Weirdly, using the suggested config in the docs doesn't run PHP at all, while running my own version at least allows the frontpage to render, but the dashboard is completely inaccessible. Perhaps there's an up-to-date fork somewhere.~~ Turns out Cachet doesn't like `=404` at the end of the `try_files` directive.

TL;DR someone needs to update Cachet to support the latest versions of nginx and PHP, among other things.

After a few hours of troubleshooting, I've managed to get Cachet 2.4 functional under nginx and PHP7.4. I'm assuming PHP8 breaks several things.

**Comment by j-a-pope** [comment]
@elderlabs I have created a fork of Cachet called [Okazanta](https://github.com/Okazanta/Okazanta-core) and we would be really happy to take a look at any issues or any pull requests that you could come up with after your experiences of getting it running.

**Comment by jbrooksuk** [comment]
Hey 👋 

Please check out https://github.com/CachetHQ/Cachet/discussions/4342

TLDR; Cachet is back! 🎉
